### PR TITLE
[wasm][aot] Enable System.Runtime.Tests

### PR DESCRIPTION
**Pull request by radical**
.. this hasn't crashed in last 14 days on the tracking PR.
- And disable the one failing test with ActiveIssue

### CODE REVIEW

File: src/libraries/System.Runtime/tests/System/Type/TypeTests.cs [hunk 1]
```diff
@@ -528,6 +528,7 @@ public void GetTypeByName_Invalid(string typeName, Type expectedException, bool
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/52393", typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser), nameof(PlatformDetection.IsMonoAOT))]
```
Comment (lewing) [inline src/libraries/System.Runtime/tests/System/Type/TypeTests.cs:531]: is this going to fail on any mono AOT?

**Review by lewing — APPROVED** [review]
Looks ok, one question about the platform annotation

File: src/libraries/System.Runtime/tests/System/Type/TypeTests.cs (hunk 1 above)
Comment (radical) [reply src/libraries/System.Runtime/tests/System/Type/TypeTests.cs:531]: @vargaz ^

Comment (vargaz) [reply src/libraries/System.Runtime/tests/System/Type/TypeTests.cs:531]: It might, Type.GetType () might need a stack walk to find the 'current' assembly.

**Pull request merged**